src/main/java/mate/academy/internetshop/service/BuckedService.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.internetshop.service;
+
+import java.util.List;
+import mate.academy.internetshop.model.Bucket;
+
+public interface BuckedService {
+    Bucket addItem(Bucket bucket, Long itemId);
```

Comment:
Відповідно до наступного завдання має бути так:
```suggestion
    Bucket addItem(Bucket bucket, Item item);
```

Reply:
Done